data/tlg0062/tlg001/tlg0062.tlg001.perseus-grc2.xml

```diff
@@ -30,14 +32,14 @@
   <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
  </availability>
 </publicationStmt>
- 
+
 <sourceDesc>
 <listBibl>
 <biblStruct>
 <monogr>
 <author>Lucian</author>
 <title>Lucian</title>
-<editor>A. M. Harmon</editor>
+<editor>Austin Morris Harmon, 1878-1950</editor>
```

Comment:
As above, I would revert to A. M. Harmon, but the years should not be here.

Reply:
@AlisonBabeu ??

Reply:
I don't think we've ever put the years here, but at least it should be an easy one to take out! :)